sonar-plugin-api/src/main/java/org/sonar/api/measures/Metric.java

```diff
@@ -20,6 +20,7 @@
 package org.sonar.api.measures;
 
 import com.google.common.base.Function;
+import com.google.common.base.Objects;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.google.common.base.Objects'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUselessImportCheck)